packages/renderless/src/tabs-mf/index.ts

```
@@ -152,7 +152,12 @@ export const removeItem =
 
       if (isCurrent) {
         const nextName = nextNav ? nextNav.name : state.items[0]?.name || ''
-        api.changeCurrentName(nextName)
+        if (silent) {
+          // 弹窗关闭等 teardown 场景：仅同步内部状态，不触发 modelValue/activeName，避免关闭弹窗时误触发父组件 v-model 变更
+          state.items.forEach((item) => (item.selected = item.name === nextName))
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)